index.bs

```diff
@@ -721,7 +744,7 @@ partial interface ModelBuilder {
             builder.mul(
               r,
               builder.add(
-                (recurrentBias ? builder.slice(recurrentBias, [2 * hiddenSize], [hiddenSize]) : zero),
+                (options.recurrentBias ? builder.slice(options.recurrentBias, [2 * hiddenSize], [hiddenSize]) : zero),
                 builder.matmul(
                   hiddenState, 
                   builder.transpose(builder.slice(recurrentweight, [2 * hiddenSize, 0], [hiddenSize, -1]))
```

Comment:
There is a old typo, the `recurrentweight` should be `recurrentWeight` in this pseudo code. I found that in recent implementation of gru in webnn-polyfill.

Reply:
Fixed.